nordicsemi/dfu/bluez.py

```diff
@@ -0,0 +1,748 @@
+import binascii
+import functools
+import threading
+import time
+from uuid import UUID
+
+import dbus
+import dbus.types
+import dbus.mainloop.glib
+from gi.repository import GLib
+from dbus.exceptions import DBusException
+
+from pc_ble_driver_py.ble_driver import *
+from pc_ble_driver_py.exceptions import NordicSemiException
+from pc_ble_driver_py.observers import *
+
+logger  = logging.getLogger(__name__)
+
+
+DBUS_OBJECT_MANAGER_IF = "org.freedesktop.DBus.ObjectManager"
```

Comment:
Is `IF` from interface? If yes maybe sth like this would be better: `IDBUS_OBJECT_MANAGER`